index.html

```diff
@@ -320,27 +422,35 @@ <h3 id="placement-of-mono-ruby">Placement of mono-ruby</h3>
   <li>
     <p>When the ruby string is longer than the base character string,
     the part of the ruby string that extends beyond the base characters
-    must not hang over the characters preceding or following,
-    if they are
-    <a href="https://www.w3.org/TR/jlreq/#cl-19">ideographic characters (cl-19)</a>,
-    <a href="https://www.w3.org/TR/jlreq/#cl-15">Hiragana (cl-15)</a>,
-    <a href="https://www.w3.org/TR/jlreq/#cl-26">Katakana (cl-16)</a>,
-    etc [[JLREQ]].
+    must not hang over the characters preceding or following 
+    (see <a href="#mono-ex-1"></a>).
     Space is introduced accordingly
     between these preceding or following characters and the base characters.
-    <aside class=note title="Protruding over surrounding characters">
-      The main placement method defined in JIS X 4051 [[JIS4051]]
-      allows some amount of overhang over the preceding and following base characters,
-      but recognizes the method defined here as an allowed variant.
-    </aside>
     <figure id=mono-ex-1>
-      <img src="img/fig5.svg" />
+      <img src="img/fig08.svg" />
       <figcaption>Example 1 of mono-ruby protruding</figcaption>
     </figure>
-    However, in the following cases,
+
+    However, in the following punctuation marks 
+    like <a href="https://www.w3.org/TR/jlreq/#cl-06">Full stops (cl-06)</a>
+    [[JLREQ]]
+    which have spacing before or after the symbols, 
     the ruby characters do hang over the preceding or following characters
     (see <a href="#mono-ex-2"></a>).
-    
+    <span class="checkme">
+    (Punctuation marks like 
+    <a href="https://www.w3.org/TR/jlreq/#cl-06">Full stops (cl-06)</a> 
+    [[JLREQ]] play an important role as breaks between sentences, 
+    it is desired to keep constant spacing for preceding or following of these 
+    characters that having extra spacing around these characters could change 
+    a meaning of breaks between sentences. 
+    Also there is no issue like ones noted in note 
+    "Protrusion of ruby from base characters". 
```

Comment:
We can add a link to this note ("Protrusion of ruby from base characters").

Reply:
will be done as a part of #22